WooCommerce/Classes/ViewRelated/Products/Add Product/AddProductWithAI/StartingInfo/ProductCreationAIStartingInfoView.swift

```diff
@@ -0,0 +1,177 @@
+        ScrollView {
+            VStack(alignment: .leading, spacing: 0) {
+                VStack(alignment: .leading, spacing: Layout.titleBlockSpacing) {
+                    // Title label.
+                    Text(Localization.title)
+                        .fontWeight(.bold)
+                        .titleStyle()
+
+                    // Subtitle label.
+                    Text(Localization.subtitle)
+                        .foregroundColor(Color(.secondaryLabel))
+                        .bodyStyle()
+                }
+                .padding(.bottom, Layout.titleBlockBottomPadding)
+
+                VStack(alignment: .leading, spacing: Layout.titleBlockSpacing) {
+                    VStack(alignment: .leading, spacing: Layout.editorBlockSpacing) {
+                        VStack(spacing: 0) {
+                            ZStack(alignment: .topLeading) {
+                                TextEditor(text: $viewModel.features)
+                                    .bodyStyle()
+                                    .foregroundColor(.secondary)
+                                    .padding(insets: Layout.messageContentInsets)
+                                    .frame(minHeight: Layout.minimumEditorHeight, maxHeight: .infinity)
+                                    .focused($editorIsFocused)
+
+                                // Placeholder text
+                                placeholderText
+                            }
+
+                            Divider()
+                                .frame(height: Layout.dividerHeight)
+                                .foregroundColor(Color(.separator))
+
+                            readTextFromPhotoButton
+                                .padding(insets: Layout.readTextFromPhotoButtonInsets)
+                        }
+                        .overlay(
+                            RoundedRectangle(cornerRadius: Layout.cornerRadius).stroke(editorIsFocused ? Color(.brand) : Color(.separator))
+                        )
+                    }
+                }
+            }
+            .padding(insets: Layout.insets)
+        }
+        .safeAreaInset(edge: .bottom) {
+            VStack {
+                // CTA to continue to next screen.
+                continueButton
+                    .padding()
+            }
+            .background(Color(uiColor: .systemBackground))
+        }
+    }
+}
+
+private extension ProductCreationAIStartingInfoView {
+    var readTextFromPhotoButton: some View {
+        HStack {
+            Button {
+                viewModel.didTapReadTextFromPhoto()
+                // TODO: Launch photo selection flow
+            } label: {
+                HStack(alignment: .center, spacing: Layout.UsePackagePhoto.spacing) {
+                    Image(systemName: Layout.UsePackagePhoto.cameraSFSymbol)
+                        .renderingMode(.template)
+                        .fontWeight(.semibold)
+                        .foregroundColor(Color(.brand))
```

Comment:
Super nit: I'd suggest using `Color.accentColor` just in case we decide to change the accent color or brand color to be different. We should use brand color only for components that are directly related to our branding IMO.